packages/fluentui/react-bindings/src/hooks/useCSS.ts

```diff
@@ -0,0 +1,94 @@
+import { serializeStyles } from '@emotion/serialize';
+import { ICSSInJSStyle, ThemePrepared } from '@fluentui/styles';
+import cx from 'classnames';
+// @ts-ignore
+import _Stylis from 'stylis';
+// @ts-ignore No typings :(
+import focusVisiblePlugin from '@quid/stylis-plugin-focus-visible';
+// @ts-ignore No typings :(
+import rtlPlugin from 'stylis-plugin-rtl';
+
+import { useFluentContext } from '../context';
+
+//
+// Types
+//
+
+export type UseCSSStyle = Omit<ICSSInJSStyle, 'animationName'>;
```

Comment:
Do we support our animation definition in `useCss`? If not we do not have to omit this prop.

Reply:
No we don't support, changed to:

```tsx
// Inline keyframe definitions are not supported by useCSS() hook
export type UseCSSStyle = Omit<ICSSInJSStyle, 'animationName'> & { animationName?: string };
```